### ISSUE TITLE
Mac and iOS App

### ISSUE THREAD

**Issue by fab1ton** [nextcloud/passman#358]
Will you be brining an iOS and OSX app as well?

**Comment by brantje** [comment]
It's planned. But no ETA because we're doing this in our free time.

**Comment by fab1ton** [comment]
Great news! Keep up the great work. Looking forward to it!

**Comment by mrdmiller2** [comment]
Anything we can do to speed it up?.. Donate?

**Comment by eldon** [comment]
+1 Anything we can do to support? Donate? Contribute free dev time? :)

**Comment by realtek2017** [comment]
I have iOS dev experience, how can we contribute? Thanks

**Comment by rsawyer6003** [comment]
This would solve a problem I have now, I'd be willing to $$$upport the development of this. I have no iOS dev experience so all I can do is chip in cash.

**Comment by brantje** [comment]
@eldon @realtek2017 Do you guy's still like to work on this? We're getting an iPhone 5s soon, so we might get something started soon =).

**Comment by Zjemm** [comment]
nice work

**Comment by kbaegis** [comment]
Yes please! +1

**Comment by nr458h** [comment]
+1

**Comment by Y0nnyy** [comment]
@brantje i started an ios version by myself in react native https://github.com/Y0nnyy/passman-ios but it is far from finished. But you can take a look.

**Comment by gt2416** [comment]
Any official updates on this ?

**Comment by Y0nnyy** [comment]
@gt2416 You can view my current state and progress under [passman-ios/projects](https://github.com/Y0nnyy/passman-ios/projects). Some features are missing for first release but it will be ready in the near future. Also it will be released under passman and not by myself. Maybe @brantje can give a more official answer :)?

**Comment by mitras2** [comment]
@Y0nnyy 
Any chance we will see a passman for iOS in the near future? Do you new help with the development? I have no iOS dev experience but I'm ok in Typescript, JS, CSS and all this Web stuff.

**Comment by Y0nnyy** [comment]
@mitras2 I reached out to the nextcloud team to publish the app to the appstore under their developer organization. They promised me that this would be okay but i haven't got their another answer despite their promise. So i could ask them once again but apparently it is not their goal to to publish this app. 

Also i am using another password manager and have not much time to continue developig the app. I would appreciate your help. I added some isses and prioritized them on those project boards. Just pick one and submit a pull request. 
If you have any issue with react-native or to get it run/build, just message me or submit an issue.

**Comment by kbaegis** [comment]
@Y0nnyy Do you think it pertains to all of apple’s restrictions on the appstore? You may have to publish and sign it yourself; if the NextCloud team takes it on they’d have to audit and incorporate your code into their CI and maintain it which I would expect to take a little time.

**Comment by Y0nnyy** [comment]
@kbaegis I don't thinks aplle's restrication are the reason. I think they forgot about it and it isn't that important for them. Or there are reasons that they don't want to publish it under their name but the didn't tell me about it. 
And i don't want to afford the apple dev account costs because if i publish the app i want it to make free. So we are kind of stuck on this subject.

**Comment by adrianolucas** [comment]
@Y0nnyy, is the code completed/stable for this app? If so, we could publish it under our company account and make it available for free on the App Store. Let me know your thought on that.

**Comment by adrianolucas** [comment]
Above message sent to @kegesh, not @Y0nnyy...

**Comment by kegesch** [comment]
@adrianolucas The App is stable (at least it was the last time it tested it) but is far from finished (see the project's board).
Generally i would be fine with this idea, but first i would like to know what kind if company it is. You are free to send me a private mail :)

**Comment by adrianolucas** [comment]
@kegesh, sure, that's fine. How to contact you in private? I've visited your main page on GitHub but couldn't find any private messaging.

**Comment by adrianolucas** [comment]
@kegesh, contact me through http://adrianolucca.me. Cheers.

**Comment by adrianolucas** [comment]
@kegesh, I've just replied to your email. Let me know if you have not received it.

**Comment by e-alfred** [comment]
@adrianolucas Anything new about publishing the app? It would be awesome if you can do that.

**Comment by e-alfred** [comment]
@kegesch Maybe @jospoortvliet can help you with publishing this app through Nextcloud?

**Comment by nefh** [comment]
Hi, any updates on iOS app?

**Comment by kegesch** [comment]
No, currently i am looking for someone who will continue the development since i stopped using Passman myself and the development of the iOS app.

**Comment by purefan** [comment]
Can we tempt someone with a higher bounty? Im willing to chip in, especially now that iOS allows external password managers

**Comment by nefh** [comment]
I've already donated, but if somebody would like to really take on the ios app, then I could donate even more.

**Comment by e-alfred** [comment]
@purefan @nefh As the money is only paid out if the job is done by Bountysource, you can pledge any amount which will definitely give people an incentive to do it, especially considering the costs to develop for MacOS/iOS.
